packages/kbn-optimizer/limits.yml

```diff
@@ -7,7 +7,7 @@ pageLoadAssetSize:
   banners: 17946
   bfetch: 22837
   canvas: 1066647
-  cases: 170000
+  cases: 175000
```

Comment:
I'm temporarily raising the bundle size here so we can get this merged. We still have this issue: https://github.com/elastic/kibana/issues/103748 and we're planning on working on it. We'd just like to get this PR merged in 8.8 and we won't be able to realistically lower the bundle size in this release.